packages/js/src/first-time-configuration/first-time-configuration-steps.js

```diff
@@ -14,13 +14,7 @@ import IndexationStep from "./tailwind-components/steps/indexation/indexation-st
 import SiteRepresentationStep from "./tailwind-components/steps/site-representation/site-representation-step";
 import PersonalPreferencesStep from "./tailwind-components/steps/personal-preferences/personal-preferences-step";
 import FinishStep from "./tailwind-components/steps/finish/finish-step";
-
-const STEPS = {
-	optimizeSeoData: "optimizeSeoData",
-	siteRepresentation: "siteRepresentation",
-	socialProfiles: "socialProfiles",
-	personalPreferences: "personalPreferences",
-};
+import { STEPS } from "./tailwind-components/steps/constants";
```

Comment:
uhm, I do not agree with placing it in `tailwind-components`, I see this more related to the overall structure of the FTC more than to the individual steps... but it's just a side note

Reply:
I see your point, I'll move it to the root FTC folder (I see that's the go-to location for our `constants.js` files).